src/python/dxpy/bindings/dxfile.py

```diff
@@ -402,6 +445,27 @@ def write_request(data_for_write_req):
             # performance when len(data) >> _write_bufsize
             self.write(data[remaining_space:], **kwargs)
 
+    def _readable_part_size(bytes):
+        """
+            Returns the file size in readable form'
+        """
+        B = float(bytes)
+        KB = float(1024)
+        MB = float(KB * 1024)
+        GB = float(MB * 1024)
+        TB = float(GB * 1024)
+
+        if B < KB:
+            return '{0} {1}'.format(B, 'Bytes' if 0 == B > 1 else 'Byte')
```

Comment:
This desugars to (0 == B and B > 1) which never holds. I think you meant to write "or".